### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by zeldapool-ada-pool**
Added Z3LDA pool

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -12,6 +12,20 @@
         "telegram": "AdaStak_Official",                              
         "operatorName": "Will",		                                   
         "github": "wrp0"                                                      
+    },
+    {
+        "ticker": "Z3LDA",                                                      
+        "poolId": "263578cc21e4acdfb86ca47a90fbb9200a203e45975e6fea423fb438",
+        "social": {                                                            
+            "twitter": "Z3ldaP",
+            "website": "https://www.facebook.com/z3ldastakingpool",
```
Comment (Bez625) [inline registry.json:21]: This should be your personal website - please leave empty or add a website

File: registry.json [hunk 2]
```diff
@@ -12,6 +12,20 @@
         "telegram": "AdaStak_Official",                              
         "operatorName": "Will",		                                   
         "github": "wrp0"                                                      
+    },
+    {
+        "ticker": "Z3LDA",                                                      
+        "poolId": "263578cc21e4acdfb86ca47a90fbb9200a203e45975e6fea423fb438",
+        "social": {                                                            
+            "twitter": "Z3ldaP",
```
Comment (Bez625) [inline registry.json:20]: This account has been suspended. If you no longer plan on using it please can you remove from the PR?

File: registry.json [hunk 3]
```diff
@@ -12,6 +12,20 @@
...
+            "website": "https://www.facebook.com/z3ldastakingpool",
+            "facebook": "https://www.facebook.com/z3ldastakingpool",  
+            "instagram": " ",
+            "youtube": " "
+        },
+        "telegram": "z3ldastakepooll",
```
Comment (Bez625) [inline registry.json:26]: This could not be found. It should be a users telegram handle - not the name of a group / page.

File: registry.json [hunk 4]
```diff
@@ -12,6 +12,20 @@
...
+        "poolId": "263578cc21e4acdfb86ca47a90fbb9200a203e45975e6fea423fb438",
+        "social": {                                                            
+            "twitter": "Z3ldaP",
+            "website": "https://www.facebook.com/z3ldastakingpool",
+            "facebook": "https://www.facebook.com/z3ldastakingpool",  
+            "instagram": " ",
+            "youtube": " "
+        },
+        "telegram": "z3ldastakepooll",                              
+        "operatorName": "Beej",
```
Comment (Bez625) [inline registry.json:27]: Please can you change to your real first name, real full name or leave blank?

File: registry.json [hunk 5]
```diff
@@ -12,6 +12,20 @@
...
         "operatorName": "Will",		                                   
         "github": "wrp0"                                                      
+    },
+    {
+        "ticker": "Z3LDA",                                                      
+        "poolId": "263578cc21e4acdfb86ca47a90fbb9200a203e45975e6fea423fb438",
+        "social": {                                                            
+            "twitter": "Z3ldaP",
+            "website": "https://www.facebook.com/z3ldastakingpool",
+            "facebook": "https://www.facebook.com/z3ldastakingpool",
```
Comment (Bez625) [inline registry.json:22]: There is no content on this fb page - please could you show some activity?

**Review by Bez625 — CHANGES_REQUESTED** [review]
Please address the comments

**Comment by Bez625** [comment]
@zeldapool-ada-pool Please can you address the comments if you still want to be added?

**Comment by Bez625** [comment]
@zeldapool-ada-pool if we dont hear from you soon we will close the PR

**Comment by Bez625** [comment]
Closing as there has been no update recently. Please open a new PR if you wish to be added

**Comment by zeldapool-ada-pool** [comment]
Apologies I’ve been snowed under. Thanks

**Pull request closed without merging**